src/Uno.UWP/Graphics/Imaging/BitmapEncoder.cs

```diff
@@ -0,0 +1,28 @@
+﻿#if !__NETSTD_REFERENCE__
```

Comment:
Is there a reason to exclude `__NETSTD_REFERENCE__` ? This "platform" is not used, it's just a matter of contract while developing, you can just let that compile also on it.